agent/main.go

```diff
@@ -32,26 +33,19 @@ import (
 	"github.com/sirupsen/logrus"
 	"golang.org/x/sync/errgroup"
 	"golang.org/x/sys/unix"
+
+	"github.com/firecracker-microvm/firecracker-containerd/eventbridge"
 )
 
 const (
-	defaultPort = 10789
+	defaultPort      = 10789
+	defaultNamespace = "default"
```

Comment:
This is defined in `namespaces.Default`.

Reply:
Fixed while squashing commits